src/utils/languages.js

```diff
@@ -1486,3 +1486,20 @@ export const setAppLanguage = lang => {
 export const getAppLanguage = () => {
   return localStorage.getItem('language-app')
 }
+
+export const setDeviceLanguage = () => {
+  const navigatorLang = navigator.language
+  const lang = navigatorLang.substr(0, navigatorLang.indexOf('-'))
```

Comment:
Those 2 lines are repeated, they can be abstracted as `getCurrentDeviceLanguage`